### DISCUSSION TITLE
How to delete a photo

### DISCUSSION THREAD

**Discussion by sse450** [immich-app/immich#4884] [Q&A] [answered]
I am new to immich. Thanks to everyone who created such a nice software.

I know, it sounds very dumb. But, I don't understand how to delete a photo. I delete the photo on the web interface, but it still exists in library. Is it by design? If not, how can I remove it from everywhere including filesystem. 

By the way, immich is self hosted and there is no external library. All photos are uploaded from the web interface.

Thanks

**Comment by bo0tzz** [comment, marked as answer]
After you delete an asset it goes into the trash, which you can find in the left menu column. If you delete it from the trash then it is removed from disk as well.

**Reply by sse450** [reply]
Doh! So easy.
Thank you.
